portfolio/src/main/webapp/script.js

```diff
@@ -1,4 +1,63 @@
-
 function scrollUp() {
-    window.scrollTo({top:0, behavior: 'smooth'});
-}
\ No newline at end of file
+  window.scrollTo({top: 0, behavior: 'smooth'});
+}
+
+async function printComments() {
+  let limit = document.getElementById('limit').value;
+
+  fetch('/comments?limit=' + limit)
+      .then(response => response.json())
+      .then((data) => {
+        let allComments = document.getElementById('allComments');
+
+        while (allComments.lastChild) {
+          allComments.removeChild(allComments.lastChild)
+        }
+
+        data.forEach((comment) => {
+          createCommentElement(comment);
+        });
+      });
+}
+
+
+function createCommentElement(comment) {
+  const allComments = document.getElementById('allComments');
+  const commentDiv = document.createElement('div');
+
+  // name
+  const nameElement = document.createElement('p');
+  let nameTextNode = document.createTextNode(comment.name + ' left a comment:');
+  nameElement.appendChild(nameTextNode);
+  commentDiv.appendChild(nameElement);
+
+  // comment
+  const commentElement = document.createElement('p');
+  let commentTextNode = document.createTextNode(comment.text);
+  commentElement.appendChild(commentTextNode);
+  commentDiv.appendChild(commentElement);
+
+  // time
+  const timeEl = document.createElement('p');
+  let timeTextNode = document.createTextNode('at ' + comment.timestamp);
+  timeEl.appendChild(timeTextNode);
+  commentDiv.appendChild(timeEl);
```

Comment:
Lots of duplication here. Might be worth creating an "appendComment" method that does this for you, and takes the text to add as a parameter.